### ISSUE TITLE
accessibility: SameDiff results not readable

### ISSUE THREAD

**Issue by rahulbot** [c4fcm/DataBasic#179]
Rich says:
"
This tool produces output that is difficult to read.
 Each of the three lists are presented as one long string of characters with no interveening space
We suggest:
- present the wordcloud as is done now, but add aria-hidden="true" to the container div;
- add a simple html list containing the words in order (most to least frequent) for each cloud and move these off-screen via css clip;

"
